tests/Data/Food/PreparationTest.elm

```diff
@@ -23,8 +23,7 @@ suite =
                   , applyRawToCookedRatio = False
                   }
                     |> Preparation.apply builderDb (Mass.kilograms 1)
-                    |> Result.map (Impact.getImpact (Impact.trg "cch") >> Unit.impactToFloat)
-                    |> Result.withDefault 0
+                    |> (Impact.getImpact (Impact.trg "cch") >> Unit.impactToFloat)
                     |> Expect.within (Expect.Absolute 0.001) 0.08
```

Comment:
Nit: maybe just me but I'd find the traditional chaining more legible:

```elm
    |> Preparation.apply builderDb (Mass.kilograms 1)
    |> Impact.getImpact (Impact.trg "cch")
    |> Unit.impactToFloat
    |> Expect.within (Expect.Absolute 0.001) 0.08
```

Reply:
Indeed, makes more sense, thanks for pointing it out!